### PR TITLE
Create unbricking.md

### PR DESCRIPTION
**Pull request by AlessandroAU**

### CODE REVIEW

File: docs/quick-start/unbricking.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Now, ALL the ESP8285 based RX’s have a factory installed bootloader onboard which CANNOT be damaged or tampered with. You can completely screw up the upgrade and always get it back using this default bootloader. 
+
+First, we must stop betaflight getting in the way and screwing things up with telemetry. Load up the configurator, go to the configuration page and then disable telemetry. Press save and reboot. If not already done so, the RX must be wired to the Flight controller or USB-SERIAL adapter. 
+
+Take note that the RX wires goes to the TX and vice versa, BOTH wires must be connected. If your receiver was already working properly and it’s still wired in you’ve already done this step.
+
+Next, we must activate the ROM bootloader. On every ESP82585 based RX there is a GPIO0 pin somewhere on the board, it may be connected to a button, or it may be a bare pad on the PCB. You must connect this pin to ground, THEN apply power. If you’ve done it right the LED should be solid on. One power is applied you may stop holding the pin to ground. How exactly this is done depends on the type of receiver.
```
Comment (JyeSmith) [inline docs/quick-start/unbricking.md:19]: ESP82585... extra number

Comment (JyeSmith) [reply docs/quick-start/unbricking.md:19]: One power = Once power

File: docs/quick-start/unbricking.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+If you have a receiver with a button, just hold it down while you apply power to enter the bootloader. However, if you have an EP1/EP2 you have to short the boot pad which is located just above GND to GND with solder. 
+
+### Happymodel EP1, EP2 boot pads
+
+![HM2400 connection](../assets/images/hm2400.png)
+
+One the receiver is in bootloader mode, all you need to do is configure the RX in the configurator as per normal and then upload with the ‘via_BetaflightPassthrough’ method. If all goes well, you should see it upload successfully.
```
Comment (JyeSmith) [inline docs/quick-start/unbricking.md:27]: One the receiver  = Once the receiver

**Comment by JyeSmith** [comment]
I would recommend soldering the pad to gnd, and even holding the button down until upload has started.  Ive had it not work a number of times when only holding during power up.

**Comment by maybenikhil** [comment]
Can this be updated and paralleled into development as well

**Comment by AlessandroAU** [comment]
Yeah lets think about and not merge for now

**Pull request merged**